### PR TITLE
refactor: eslint fixes

### PR DESCRIPTION
**Pull request by saikumarrs**
## Description of the change

- eslint auto-fixes
- Also, some eslint suggestions were incorporated.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

N/A

### Development

- [x] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

---
This change is [Reviewable](https://reviewable.io/reviews/rudderlabs/rudder-sdk-js/578)

### CODE REVIEW

File: integrations/Clevertap/browser.js [hunk 1]
```diff
@@ -117,7 +117,6 @@ class Clevertap {
     Object.keys(payload).map((key) => {
       if (isObject(payload[key])) {
         logger.debug("cannot process, unsupported traits");
-        return;
```
Comment (MoumitaM) [inline integrations/Clevertap/browser.js:120]: Why return statements are removed?

File: integrations/Qualtrics/browser.js [hunk 2]
```diff
@@ -47,36 +47,36 @@ class Qualtrics {
...
               d = d.substring(1, d.length);
-            if (0 == d.indexOf(a)) return d.substring(a.length, d.length);
+            if (d.indexOf(a) == 0) return d.substring(a.length, d.length);
           }
           return null;
         };
         this.set = function (a, c) {
-          var b = "",
-            b = new Date();
+          var b = "";
```
Comment (MoumitaM) [inline integrations/Qualtrics/browser.js:64]: No need for this line no 64.

File: utils/xhrModule.js [hunk 3]
```diff
@@ -90,9 +89,6 @@ class XHRQueue {
               )
             );
           } else {
-            // logger.debug(
```
Comment (MoumitaM) [inline utils/xhrModule.js:93]: Add some comments to identify this is the successful case

File: integrations/Clevertap/browser.js (hunk 1 above)
Comment (saikumarrs) [reply integrations/Clevertap/browser.js:120]: It's an eslint fix. It doesn't make sense to have an empty return statement.

File: utils/xhrModule.js (hunk 3 above)
Comment (saikumarrs) [reply utils/xhrModule.js:93]: It didn't make sense to log the success case. Hence, I removed even the previously commented-out log statement.

File: integrations/Qualtrics/browser.js (hunk 2 above)
Comment (saikumarrs) [reply integrations/Qualtrics/browser.js]: Done.

File: utils/linker/crc32.js [hunk 4]
```diff
@@ -10,12 +10,12 @@
  * @returns array of CRC table
  */
 
-const makeCRCTable = () => {
+const makeCRCTable = function () {
   const crcTable = [];
   let c;
-  for (let n = 0; n < 256; n += 1) {
```
Comment (MoumitaM) [inline utils/linker/crc32.js]: Why change the syntax? Both do the same thing.

File: integrations/VWO/browser.js [hunk 5]
```diff
@@ -148,8 +148,7 @@ class VWO {
     const eventName = rudderElement.message.event;
     if (eventName === 'Order Completed') {
       const total = rudderElement.message.properties
-        ? rudderElement.message.properties.total ||
-          rudderElement.message.properties.revenue
+        ? rudderElement.message.properties.total || rudderElement.message.properties.revenue
```
Comment (MoumitaM) [inline integrations/VWO/browser.js:151]: Shouldn’t there be a condition?

File: integrations/Posthog/browser.js [hunk 6]
```diff
@@ -55,29 +55,24 @@ class Posthog {
         (e._i = []),
         (e.init = function (i, s, a) {
           function g(t, e) {
-            const o = e.split(".");
```
Comment (MoumitaM) [inline integrations/Posthog/browser.js]: Multiple syntax differences across this file. Please take look.

File: integrations/Qualtrics/browser.js [hunk 7]
```diff
@@ -43,32 +43,24 @@ class Qualtrics {
...
-          for (
-            var a = `${a}=`,
-              c = document.cookie.split(";"),
-              b = 0,
-              e = c.length;
-            b < e;
-            b++
-          ) {
-            for (var d = c[b]; d.charAt(0) == " "; )
-              d = d.substring(1, d.length);
```
Comment (MoumitaM) [inline integrations/Qualtrics/browser.js]: Syntax difference. Please take look.

File: integrations/SnapPixel/browser.js [hunk 8]
```diff
@@ -65,11 +65,11 @@ class SnapPixel {
         a.handleRequest ? a.handleRequest.apply(a, arguments) : a.queue.push(arguments);
       });
       a.queue = [];
-      const s = "script";
```
Comment (MoumitaM) [inline integrations/SnapPixel/browser.js]: Syntax difference here also.

**Review by MoumitaM — CHANGES_REQUESTED** [review]

File: utils/beaconQueue.js [hunk 9]
```diff
@@ -82,14 +82,11 @@ class BeaconQueue {
...
     const data = { batch };
     const payload = JSON.stringify(data, replacer);
-    const blob = new Blob([payload], { type: "text/plain" });
-    const isPushed = navigator.sendBeacon(
-      `${this.url}?writeKey=${this.writekey}`,
-      blob
-    );
-    if (!isPushed) {
-      logger.error("Unable to queue data to browser's beacon queue");
-    }
```
[the remark this answers is not in the document]
Comment (saikumarrs) [reply utils/beaconQueue.js]: The first statement is not deleted.
Restored the commented log statement.

File: utils/linker/crc32.js (hunk 4 above)
Comment (saikumarrs) [reply utils/linker/crc32.js]: `eslint` recommendation.

File: integrations/VWO/browser.js (hunk 5 above)
Comment (saikumarrs) [reply integrations/VWO/browser.js]: Sorry, didn't get you.

File: integrations/Posthog/browser.js (hunk 6 above)
Comment (saikumarrs) [reply integrations/Posthog/browser.js]: You mean w.r.t quotes around strings?

File: utils/linker/crc32.js (hunk 4 above)
Comment (MoumitaM) [reply utils/linker/crc32.js]: Okay

File: utils/xhrModule.js (hunk 3 above)
Comment (MoumitaM) [reply utils/xhrModule.js:93]: Please add a comment to understand after the request is successful it will come to this block.

**Review by MoumitaM — COMMENTED** [review]
Please address the comment in xhrModule.js file

File: utils/xhrModule.js (hunk 3 above)
Comment (saikumarrs) [reply utils/xhrModule.js:93]: Why do we need logs for a success case? It makes more sense for errors, right?

**Review by MoumitaM — APPROVED** [review]

**Pull request merged**